apps/api/src/campaign/campaign.service.ts

```diff
@@ -408,10 +410,25 @@ export class CampaignService {
         { create: { campaignId, currency: campaign.currency, name: campaign.title } }
 
     // Find donation by extPaymentIntentId and update if status allows
+    const donationNotificationSelect = {
+      id: true,
+      status: true,
+      currency: true,
+      amount: true,
+      createdAt: true,
+      extPaymentMethodId: true,
+      person: {
+        select: {
+          firstName: true,
+          lastName: true,
+          picture: true,
+        },
+      },
```

Comment:
Could we declare the DonationNotificationSelect as a type in the NotificationService and then include it here from it?